src/reducers/jobInfoReducer.test.js

```diff
@@ -1,13 +1,14 @@
 import jobInfoReducer from './jobInfoReducer.js'
 
+// need to test line 18; a break statement?
+
```

Comment:
I missed this test so I can add it. I guess it will be testing if the type isn't found it should return initial state.